src/diem/testing/miniwallet/app/models.py

```diff
@@ -108,6 +110,9 @@ def balance_amount(self) -> int:
     def __str__(self) -> str:
         return "Transaction %s" % json.dumps(asdict(self), indent=2)
 
+    def get_payee_onchain_identifier(self, hrp: str) -> str:
```

Comment:
`get_payee_onchain_identifier` => `payee_account_identifier`

`account identifier` is better name as it returns the string value of account identifier.